packages/runtime/src/next/api-handler/handlers/proxy-draft-mode.ts

```diff
@@ -55,21 +57,21 @@ async function proxyDraftModeRouteHandler(
     siteVersion: MakeswiftSiteVersion.Working,
   }
 
-  draftMode().enable()
+  await enableDraftMode()
 
   const proxyUrl = request.nextUrl.clone()
   proxyUrl.searchParams.delete('x-makeswift-draft-mode')
 
   const proxyRequest = new NextRequest(proxyUrl, { headers: request.headers })
   proxyRequest.headers.delete('x-makeswift-draft-mode')
 
-  const draftModeCookie = cookies().get('__prerender_bypass')
+  const draftModeCookie = (await cookies()).get('__prerender_bypass')
   if (draftModeCookie) {
     proxyRequest.cookies.set(draftModeCookie)
     proxyRequest.cookies.set(MAKESWIFT_DRAFT_MODE_DATA_COOKIE, JSON.stringify(draftModeData))
   }
 
-  draftMode().disable()
+  await enableDraftMode(false)
```

Comment:
This is quite confusing (both _enable_ and `false` together). If passing `false` disables Draft Mode, the name of the function is completely contradictory 😅

Reply:
`enable(false)`, `enabled={false}`, or `enabled=false` are all [pretty](https://github.com/search?q=enabled%28false%29&type=code) [common](https://github.com/search?q=enabled%3D%7Bfalse%7D&type=code) [APIs](https://github.com/search?q=enabled%3Dfalse&type=code) out there, but happy to introduce a separate method for disabling if you find this confusing.

Reply:
Now that I think about it more, I actually remember being appalled by `enable(false)` the first time I saw it in Windows APIs 25 years ago, hahaha.

Reply:
I'd be ok with `disableDraftMode()` and `enableDraftMode()`, hahaha